database/postgres/pg.js

```
@@ -0,0 +1,51 @@
+const { Client } = require('pg')
+require('dotenv').config()
+
+const pgModule = {}
+let client
+
+pgModule.startPGDB = async ()=>{
+  try{
+    client = new Client({
+      host: process.env.HOST,
+      port: process.env.PORT,
+      user: process.env.USERNAME,
+      password: process.env.PASSWORD,
+      database: process.env.DATABASE
+    })
+    await client.connect()
+  }catch(err){
+    console.log('connection failed', err)
+  }
+}
+
+pgModule.closePGDB = async ()=>{
+  try{
+    await client.end()
+  }catch(err){
+    console.log('connection failed to closed', err)
```

this should return an error so whomever calls `closePGDB` will know that it has failed.

---

please add this as a test case